### PR TITLE
Enables caldav via reverse proxy

### PR DESCRIPTION
**Pull request by mhoffmann75**
Enables caldav via reverse proxy on /caldav/ - see https://github.com/zokradonh/kopano-docker/issues/95

### CODE REVIEW

**Comment by fbartels** [comment]
Ah nice, I must say I secretly hoped that you would pick this up, since you mentioned problems getting ical running with https.

Small comments inline.

Thanks for your pr!

File: docker-compose.yml [hunk 1]
```diff
@@ -363,10 +363,12 @@ services:
       - SERVICE_TO_START=ical
       - TZ=${TZ}
       - KCCONF_ICAL_SERVER_SOCKET=http://kopano_server:236/
+      - KCCONF_ICAL_ICAL_LISTEN=*:80
```
Comment (fbartels) [inline docker-compose.yml]: this is actually not necessary, since the service already listens on port 8080

File: web/kweb.cfg [hunk 2]
```diff
@@ -152,6 +152,14 @@
         try_duration 30s
     }
 
+    proxy /caldav/ kopano_ical:80 {
```
Comment (fbartels) [inline web/kweb.cfg]: just change port here to 8080

File: web/kweb.cfg [hunk 3]
```diff
@@ -152,6 +152,14 @@
...
+        fail_timeout 10s
+        try_duration 30s
+        transparent
+        keepalive 100
+    }
+    folderish /caldav
```
Comment (fbartels) [inline web/kweb.cfg:160]: not quite sure if this is required, but it surely cannot hurt much.

**Review by fbartels — CHANGES_REQUESTED** [review]

**Review by mhoffmann75 — COMMENTED** [review]
changed as per your suggestions

**Review by fbartels — APPROVED** [review]

**Comment by fbartels** [comment]
Thanks again

**Pull request merged**